lib/install/bootstrap/install.rb

```diff
@@ -0,0 +1,8 @@
+say "Install Bootstrap 5 with Popperjs/core"
+copy_file "#{__dir__}/bootstrap_js_files.js", "bootstrap_js_files.js"
+copy_file "#{__dir__}/application.scss", "app/assets/stylesheets/application.scss"
+
+run "yarn add bootstrap@5.1.1"
```

Comment:
I haven't used Bootstrap in quite a while but is using specific version numbers the right call here?

Seems like they'd need to be bumped regularly vs. just using @latest instead.

Reply:
@DavidColby agree! fixed!